cliquet/scripts/cliquet.py

```diff
@@ -2,10 +2,17 @@
 import argparse
 import sys
 import textwrap
+import warnings
 
 from pyramid.paster import bootstrap
 
 
+def deprecated_init(env):
+    message = '"cliquet init" is deprecated. Use "cliquet migrate" instead.'
```

Comment:
That's what Django has. ``manage.py migrate`` for everything :)

Do you suggest to separate the init process from the migration too ?